lib/java/com/google/android/material/appbar/res/values/attrs.xml

```diff
@@ -190,6 +190,16 @@
       <flag name="end" value="0x00800005"/>
     </attr>
 
+    <!-- The alignment of the title text. -->
```

Comment:
Add a description to explain that the value here should be corresponding to the enum android.text.Layout.Alignment.